tests/benchmarks/app_kernel/src/mailbox_r.c

```diff
@@ -83,15 +87,18 @@ int mailbox_get(struct k_mbox *mailbox, int size, int count, unsigned int *time)
 	k_sem_take(&SEM0, K_FOREVER);
 	t = BENCH_START();
 	for (i = 0; i < count; i++) {
-		k_mbox_get(mailbox, &Message, &data_recv, K_FOREVER);
+		return_value = k_mbox_get(mailbox,
+					  &Message,
+					  &data_recv,
+					  K_FOREVER);
 	}
 
 	t = TIME_STAMP_DELTA_GET(t);
 	*time = SYS_CLOCK_HW_CYCLES_TO_NS_AVG(t, count);
 	if (bench_test_end() < 0) {
 		PRINT_OVERFLOW_ERROR();
 	}
-	return 0;
+	ARG_UNUSED(return_value);
```

Comment:
ARG_UNUSED is intended for unused parameters to functions.
How about just call k_panic() if these APIs return an unfavorable result?